joss/paper.md

```diff
@@ -30,40 +30,40 @@ Consequently, these populations undergo evolution [@pennock2007models], and can
 This experimental paradigm --- used across biological modeling, artificial life, and evolutionary computation --- complements research done using *in vitro* and *in vivo* systems by enabling the user to conduct experiments that would be impossible in the lab or field [@dolsonDigitalEvolutionEcology2021].
 One key benefit is complete, exact observability.
 For example, it is possible to perfectly record the full set of parent-child relationships over the history of a population, yielding precise and accurate phylogenies (ancestry trees).
-This information reveals the sequences of events behind gain, loss, or maintenance of specific traits, and also facilitates making inferences about the general factors driving evolution [@dolsonInterpretingTapeLife2020;@moreno2023toward].
+This information reveals the sequences of events behind gain, loss, or maintenance of specific traits, and also facilitates making inferences about the underlying mechanisms driving a given evolutionary system [mooers1997inferring;@dolsonInterpretingTapeLife2020;@moreno2023toward].
```

Comment:
maybe “inferring underlying evolutionary dynamics”?

Reply:
"making inferences about the underlying evolutionary dynamics of a system"? (otherwise we're left with "inferring underlying evolutionary dynamics of what?")

Reply:
how about injecting "given" "(as suggested)... of a given system"?

Reply:
Yeah, I like that more